docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -7,6 +7,15 @@ The ClickHouse connector allows querying tables in an external
 query data in the databases on that instance, or combine it with other data
 from different catalogs accessing ClickHouse or any other supported data source.
 
+Requirements
+------------
+
+To connect to a ClickHouse instance, you need:
+
+* ClickHouse version 20.8 or higher.
+* Network access from the Trino coordinator and workers to the ClickHouse
```

Comment:
check in the clickhouse docs if the wording is right here 

options beyond the current "instance" imho are

* and workers to Clickhouse.
* and workers to the ClickHouse cluster.
* and workers to the Clickhouse server.

Reply:
Their docs refer to both ClickHouse servers and clusters. Updated accordingly.